File: rest/taskrouter/worker-reservations/call/example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioTaskRouterClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "{{ account_sid }}"
auth_token  = "{{ auth_token }}"
workspace_sid = "{{ workspace_sid }}"
worker_sid = "{{ worker_sid }}"
reservation_sid = "{{ reservation_sid }}"

client = TwilioTaskRouterClient(account_sid, auth_token)

# call using a reservation
reservation = client.workers(workspace_sid).get(worker_sid).reservations.update(reservation_sid,
	instruction='call',
	call_from='+15558675309',
	call_url='http://example.com/agent_answer',
	call_status_callback_url='http://example.com/agent_answer_status_callback',
	call_accept='true')
print reservation.reservation_status
print reservation.worker_name

File: rest/taskrouter/worker-reservations/list-get-example1/example-1.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.sdk.TwilioRestException;
import com.twilio.sdk.TwilioTaskRouterClient;
import com.twilio.sdk.resource.list.taskrouter.ReservationList;

public class Example { 

  // Find your Account Sid and Token at twilio.com/user/account
  private static final String ACCOUNT_SID = "{{ account_sid }}";
  private static final String AUTH_TOKEN = "{{ auth_token }}";
  private static final String WORKSPACE_SID = "{{ workspace_sid }}";
  private static final String WORKER_SID = "{{ worker_sid }}";

  public static void main(String[] args) throws TwilioRestException {
    TwilioTaskRouterClient client = new TwilioTaskRouterClient(ACCOUNT_SID, AUTH_TOKEN);

    ReservationList reservations = client.getWorker(WORKSPACE_SID, WORKER_SID).getReservations();

    for(Reservation reservation : reservations) {
        System.out.println(reservation.getReservationStatus());
        System.out.println(reservation.getWorkerName());
    }
  }
}